src/Geometry/UnstructuredGrids.jl

```diff
@@ -32,9 +32,15 @@ struct UnstructuredGrid{Dc,Dp,Tp,O,Tn} <: Grid{Dc,Dp}
     reffes::Vector{<:LagrangianRefFE{Dc}},
     cell_types::Vector,
     orientation_style::OrientationStyle=NonOriented(),
-    facet_normal=nothing) where {Dc,Dp,Tp,Ti}
-
-    cell_map = _compute_cell_map(node_coordinates,cell_node_ids,reffes,cell_types)
+    facet_normal=nothing;
+    has_affine_map=nothing) where {Dc,Dp,Tp,Ti}
+
+    if has_affine_map === nothing
+      _has_affine_map = get_has_affine_map(reffes)
+      cell_map = _compute_cell_map(node_coordinates,cell_node_ids,reffes,cell_types,_has_affine_map)
+    else
+      cell_map = _compute_cell_map(node_coordinates,cell_node_ids,reffes,cell_types,has_affine_map)
```

Comment:
else
  _has_affine_map = has_affine_map
end
cell_map = _compute_cell_map(...

Reply:
I.e move the computation of cell_map outside the if

Reply:
done in d0e67e616209fa5e8f2ddd652442e2081deb3ff5